### PR TITLE
add alias/filter fields to simple and converesion metrics

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds base_measure:filter to converesion metrics and  measure:alias and measure:filter to the simple metrics per [slack thread](https://dbt-labs.slack.com/archives/C050Z6UP4RK/p1728419028140339?thread_ts=1728419028.140339&cid=C050Z6UP4RK) and [linear](https://linear.app/dbt-labs/issue/SL-2988/check-documentation-for-missing-filter-metrics-in-dbt#comment-063f9cfc)

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-update-simple-metrics-dbt-labs.vercel.app/docs/build/conversion-metrics
- https://docs-getdbt-com-git-update-simple-metrics-dbt-labs.vercel.app/docs/build/simple

### CODE REVIEW

File: website/docs/docs/build/simple.md [hunk 1]
```diff
@@ -65,9 +69,11 @@ If you've already defined the measure using the `create_metric: true` parameter,
           name: customers # The measure you are creating a proxy of.
           fill_nulls_with: 0 
           join_to_timespine: true
+          alias: customer_count
+          filter: {{ Dimension('customer__customer_total') }} >+ 20
```
Comment (mirnawong1) [inline website/docs/docs/build/simple.md]:
```suggestion
          filter: {{ Dimension('customer__customer_total') }} >= 20
```

File: website/docs/docs/build/simple.md [hunk 2]
```diff
@@ -24,6 +24,8 @@ Note that we use the double colon (::) to indicate whether a parameter is nested
 | `type_params` | The type parameters of the metric. | Required | Dict |
 | `measure` | A list of measure inputs. | Required | List |
 | `measure:name` | The measure you're referencing. | Required | String |
+| `measure:alias` | Optional [`alias`](/reference/resource-configs/alias) used to rename the measure. | Optional | String |
```
Comment (matthewshaver) [inline website/docs/docs/build/simple.md]: i think its important to be explicit that its to rename. how about:
```suggestion
| `measure:alias` | Optional [`alias`](/reference/resource-configs/alias) to rename the measure. | Optional | String |
```

File: website/docs/docs/build/simple.md [hunk 3]
```diff
@@ -24,6 +24,8 @@ Note that we use the double colon (::) to indicate whether a parameter is nested
...
+| `measure:filter` | Optional `filter` used to apply to the measure. | Optional | String |
```
Comment (matthewshaver) [inline website/docs/docs/build/simple.md]:
```suggestion
| `measure:filter` | Optional `filter` applied to the measure. | Optional | String |
```

File: website/docs/docs/build/simple.md [hunk 4]
```diff
@@ -38,6 +40,8 @@ metrics:
     type_params: # Required
       measure: 
         name: The name of your measure # Required
+        alias: The alias used to rename the measure. # Optional
```
Comment (matthewshaver) [inline website/docs/docs/build/simple.md]:
```suggestion
        alias: The alias applied to the measure. # Optional
```

File: website/docs/docs/build/simple.md [hunk 5]
```diff
@@ -38,6 +40,8 @@ metrics:
...
+        filter: The filter used to apply to the measure. # Optional
```
Comment (matthewshaver) [inline website/docs/docs/build/simple.md]:
```suggestion
        filter: The filter applied to the measure. # Optional
```

**Comment by mirnawong1** [comment]
merging as folded in Matt's feedback

**Pull request merged**